library/src/test/java/bio/terra/landingzone/library/landingzones/management/LandingZoneManagerIntegrationTest.java

```diff
@@ -1,175 +1,25 @@
 package bio.terra.landingzone.library.landingzones.management;
 
-import static org.awaitility.Awaitility.await;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.greaterThan;
 import static org.hamcrest.Matchers.hasItem;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.notNullValue;
 
-import bio.terra.landingzone.library.landingzones.AzureIntegrationUtils;
-import bio.terra.landingzone.library.landingzones.TestArmResourcesFactory;
-import bio.terra.landingzone.library.landingzones.TestUtils;
 import bio.terra.landingzone.library.landingzones.definition.DefinitionVersion;
 import bio.terra.landingzone.library.landingzones.definition.FactoryDefinitionInfo;
 import bio.terra.landingzone.library.landingzones.definition.factories.TestLandingZoneFactory;
-import bio.terra.landingzone.library.landingzones.deployment.DeployedResource;
-import bio.terra.landingzone.library.landingzones.management.deleterules.LandingZoneRuleDeleteException;
-import com.azure.resourcemanager.AzureResourceManager;
-import com.azure.resourcemanager.resources.models.ResourceGroup;
-import java.time.Duration;
 import java.util.List;
-import java.util.UUID;
-import java.util.stream.Collectors;
-import org.awaitility.Awaitility;
-import org.hamcrest.MatcherAssert;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
-import reactor.core.publisher.Flux;
-import reactor.util.retry.Retry;
 
 @Tag("integration")
 class LandingZoneManagerIntegrationTest {
```

Comment:
We already have integration test which validate LZ creation on a higher/service level.

Reply:
It would be good to rename the test to make it clearer what it is testing.

Reply:
I think there is no need to rename this test. It still tests functionality from LandingZoneManager and it is still integration testing.

Reply:
Doesn't it only validate the list of available landing zones? I'm not clear how that is an integration test, but perhaps the terminology is confusing me.

Reply:
@sergiygetlin I'm still confused about how this is an integration test.

Reply:
+1, with the changes here it seems like this can be marked as a unit test

Reply:
Agree! You both convinced me. Will move this test in LandingZoneManagerTest.